modules/preview-react/breadcrumbs/lib/Breadcrumbs/Dropdown/Button.tsx

```diff
@@ -22,33 +21,32 @@ export interface DropdownButtonProps extends IconButtonProps {
   toggled: boolean;
 }
 
-export const DropdownButton = createComponent(IconButton)({
+export const DropdownButton = createComponent(TertiaryButton)({
   displayName: 'DropdownButton',
   Component: (
     {
       buttonIcon = relatedActionsIcon,
+      variant = undefined,
       toggled,
-      variant = 'plain',
       ...elemProps
     }: DropdownButtonProps,
     ref,
     Element
   ) => {
-    const hasPlainVariant = variant === 'plain';
+    const hasVariant = variant === undefined;
```

Comment:
Something isn't right here:

1. I don't think you need this style anymore

The additional 1px horizontal margin was to account for the `plain` variant being smaller than the others. But that isn't the case anymore. I don't think you need this line of code at all.

2. If you do need the styling, I think the logic is incorrect.

`hasVariant` should be `true` when there is a variant. This would return the opposite.

```suggestion
```

Reply:
I think we should remove this line entirely.